add-on/chrome/test/xpcshell/test_loopapi_app_version.js

```diff
@@ -0,0 +1,14 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+"use strict";
+
+const { LoopAPI } = Cu.import("chrome://loop/content/modules/MozLoopAPI.jsm", {});
+var [, gHandlers] = LoopAPI.inspect();
+
+add_task(function* test_mozLoop_npsSurveyURL() {
+  let expectedVersion = MozLoopService.addonVersion;
+  // Test GetAddonVersion
+  gHandlers.GetAddonVersion({ data: [] }, result => {
+    Assert.equal(result, expectedVersion, "should get mozLoopService addonVersion value correctly");
```

Comment:
We should be able to add a test for the stripping of the alpha value as well. If you take a look at `test_loopservice_initialize` then you should be able to do something like:

```
MozLoopService.initialise("1.3.2alpha")
gHandlers.GetAddonVersion(..)
```